lib/plug/conn.ex

```diff
@@ -653,6 +653,37 @@ defmodule Plug.Conn do
     %{conn | resp_headers: List.keystore(headers, key, 0, {key, value})}
   end
 
+  @doc ~S"""
+  Similar to `put_resp_header` this functions adds a new response header (`key`)
+  if not present, otherwise it wont replace the existing header with `key` but
```

Comment:
"wont" -> "won't".

Reply:
The first pagraph of the documentation should always be a short sentence as it is used as a summary.